src/globus_sdk/services/transfer/client.py

```diff
@@ -112,44 +113,76 @@ class TransferClient(client.BaseClient):
     scopes = TransferScopes
     default_scope_requirements = [Scope(TransferScopes.all)]
 
-    def add_app_data_access_scope(self, collection_id: UUIDLike) -> TransferClient:
+    def add_app_data_access_scope(
+        self, collection_id: UUIDLike | t.Iterable[UUIDLike]
+    ) -> TransferClient:
         """
-        Add a dependent ``data_access`` scope for a given ``collection_id`` to this
-        client's ``GlobusApp``. Useful for resolving ``ConsentRequired`` errors
+        Add a dependent ``data_access`` scope for one or more given ``collection_id``
+        to this client's ``GlobusApp``. Useful for resolving ``ConsentRequired`` errors
         when using standard Globus Connect Server mapped collections.
-        ``collection_id`` must be for a standard Globus Connect Server mapped
-        collection or app driven authentication flows will hit unknown scope errors.
+
+        ``collection_id`` must be for standard Globus Connect Server mapped
+        collections or app driven authentication flows will hit unknown scope errors.
 
         Returns ``self`` for chaining.
 
         Raises ``GlobusSDKUsageError`` if this client was not initialized with an app.
 
-        :param collection_id: UUID for a standard Globus Connect Server mapped
-            collection.
+        :param collection_id: an ID for a standard Globus Connect Server mapped
```

Comment:
Same as above...is the word "standard" needed here?

Reply:
I think I will pare this text down to `ID of a collection or an iterable of IDs.` and let the surrounding context -- primarily my suggested warning -- handle the nuance around correct usage.